macros/base/start_print.cfg

```
@@ -370,6 +370,7 @@ gcode:
     {% set FL_SIZE = printer["gcode_macro START_PRINT"].fl_size %}
     {% set BED_MESH_PROFILE = printer["gcode_macro START_PRINT"].bed_mesh_profile %}
 
+    {% set force_mesh = printer["gcode_macro _USER_VARIABLES"].force_mesh %}
```

Can you remove the variable and use this line instead?
```suggestion
    {% set force_mesh = params.FORCE_MESH|default(0) %}
```

---

> Can you remove the variable and use this line instead?
> 
> ```suggestion
> 
>     {% set force_mesh = params.FORCE_MESH|default(0) %}
> 
> ```
> 
> 

I think the params must be parse in this section of print_start and call after in bed mesh module?

    # Get all the parameters passed from the slicer

---

why not using the implementation style in start print like it is done previously all the time? the bed mesh module is never called directly outside start print